mobile-browser-based-version/server/run_server.js

```diff
@@ -1,14 +1,17 @@
 const express = require('express');
 const { ExpressPeerServer } = require('peer');
+const cors = require('cors');
 var topologies = require('./topologies.js')
 
 var myArgs = process.argv.slice(2);
 
 const app = express();
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({extended: false}));
 
 const topology = new topologies.BinaryTree()
```

Comment:
Do you plan this to be configurable per experiment in the long run?

Reply:
yes, once we have more than one topology